data/mods/Magiclysm/items/alchemy_items.json

```diff
@@ -0,0 +1,36 @@
+[
+  {
+    "id": "owlbear_stone",
+    "type": "TOOL",
+    "name": "owlbear gastrolith",
+    "description": "This strangely shaped rock was once a part of an owlbear's complex digestive system, creating a link between owl throat and bear stomach.  It is possible that it could serve a similar purpose in alchemy, helping to bind together two disparate halves.",
```

Comment:
this makes me think you want to add this to the harvest entry for owlbears as well.

Reply:
Yup. It's in the upcoming commit.